apps/builder/app/builder/features/style-panel/sections/layout/shared/flex-grid.stories.tsx

```diff
@@ -24,49 +24,232 @@ const justifyContent = [
-          <Box key={justifyContent} css={{ width: 72, fontSize: 10 }}>
-            {justifyContent}
-          </Box>
-        ))}
-      </Flex>
-      {alignItems.map((alignItems) => (
-        <Flex key={alignItems} gap={2}>
-          <Box
-            css={{
-              fontSize: 10,
-              textAlign: "right",
-              writingMode: "vertical-lr",
-              transform: "rotate(180deg)",
-            }}
-          >
-            {alignItems}
-          </Box>
+    <Flex>
+      <Box
+        css={{
+          fontSize: 10,
+          paddingTop: 200,
+          paddingLeft: 6,
+          fontWeight: "bold",
+          writingMode: "vertical-lr",
+          transform: "rotate(180deg)",
+        }}
+      >
+        align-items
+      </Box>
+      <Flex direction="column" gap={2} css={{ color: "#000" }}>
+        <Box css={{ fontSize: 10, paddingLeft: 200, fontWeight: "bold" }}>
+          justify-content
+        </Box>
+        <Flex gap={2}>
+          <Box css={{ fontSize: 10, writingMode: "vertical-lr" }}>&nbsp;</Box>
           {justifyContent.map((justifyContent) => (
-            <Box key={justifyContent} css={{ width: 72, height: 72 }}>
-              <FlexGrid
-                currentStyle={{
-                  flexDirection: {
-                    value: { type: "keyword", value: flexDirection },
-                  },
-                  justifyContent: {
-                    value: { type: "keyword", value: justifyContent },
-                  },
-                  alignItems: {
-                    value: { type: "keyword", value: alignItems },
-                  },
-                }}
-                batchUpdate={batchUpdate}
-              />
+            <Box key={justifyContent} css={{ width: 72, fontSize: 10 }}>
+              {justifyContent}
             </Box>
           ))}
         </Flex>
-      ))}
+        {alignItems.map((alignItems) => (
+          <Flex key={alignItems} gap={2}>
+            <Box
+              css={{
+                fontSize: 10,
+                textAlign: "right",
+                writingMode: "vertical-lr",
+                transform: "rotate(180deg)",
+              }}
+            >
+              {alignItems}
+            </Box>
+            {justifyContent.map((justifyContent) => (
+              <Box key={justifyContent} css={{ position: "relative" }}>
+                <FlexGrid
+                  currentStyle={{
+                    flexDirection: {
+                      value: { type: "keyword", value: flexDirection },
+                    },
+                    justifyContent: {
+                      value: { type: "keyword", value: justifyContent },
+                    },
+                    alignItems: {
+                      value: { type: "keyword", value: alignItems },
+                    },
+                  }}
+                  batchUpdate={batchUpdate}
+                />
+
+                {REFERENCE && (
```

Comment:
not sure why you have the REFERENCE, since its always a constant

Reply:
Developer may set REFERENCE to `true` to check alignment against SVGs imported from Figma.

I can just remove these references, wasn't sure they are worth to keep.

Reply:
Removed